instrumentation/kafka/kafka-clients/kafka-clients-0.11/javaagent/src/main/resources/tested_versions/kafka-clients-0.11

```diff
@@ -0,0 +1 @@
+0.11.0.0
```

Comment:
I don't see it in the "supported packages" section in the README. is it a different PR?